docs - initial doc content for Greenplum R client beta

lisakowen opened the pull request:
in this PR:
- initial docs for Greenplum R Client beta
- included some examples, plan to add more and possibly revisit the presentation of the info in future work
- added new page/topic to the Analytics subnav

notes:
- need to verify install info
- this PR does not include any reference/man pages for the functions.  currently pointing user to the html files installed with the package and the help() function.
- not sure how best to present a complete list of greenplumr functions, or if this is needed.  the PR includes a Function Summary section with a table that lists some of the functions.  thoughts?

doc review site link:  http://docs-lisa-greenplumr-beta.cfapps.io/7-0/analytics/greenplum_r_client.html

dyozie commented on gpdb-doc/dita/analytics/greenplum_r_client.xml:
```diff
@@ -0,0 +1,444 @@
+        <ul>
+          <li>The <codeph>db.gpapply()</codeph> and <codeph>db.gptapply()</codeph>
+            functions currently operate only on <codeph>conn.id = 1</codeph>.</li>
+          <li>The GreenplumR function reference pages are not yet published.</li>
```
Maybe re-point them to the github markdown pages for what's available?

lisakowen commented:
thanks for reviewing @HaozhouWang and @dyozie .  i updated the download file name, added more functions to the summary table, and made a few miscellaneous other edits.  i re-pushed the doc review site, you may need to refresh.

dyozie commented:
Thanks - those changes look good to me!
